Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml

```diff
@@ -2722,6 +2729,185 @@ script:
     - contextPath: PrismaCloudCompute.RuntimeContainerAuditEvents.provider
       description: The provider of the container.
       type: String
+  - name: prisma-cloud-archive-audit-incident
+    description: Acknowledges an incident and moves it to an archived state.
+    arguments:
+      - name: incident_id
+        description: Incident ID.
+        isArray: false
+        required: true
+      - auto: PREDEFINED
+        name: action
+        description: Action for the command. archive - incident will be archived, unarchive - incident will be unarchived.
+        predefined:
+          - "archive"
+          - "unarchive"
+        defaultValue: "archive"
+    outputs:
+      - contextPath: PrismaCloudCompute.RuntimeContainerAuditEvents.test
```

Comment:
don't see this output in the code

Reply:
there are no outputs to this command - this was a test I ran, forgot to remove it.